tests/unittests/tests-gnrc_mac_internal/Makefile.include

```diff
@@ -0,0 +1,2 @@
+USEMODULE += gnrc_priority_pktqueue
+USEMODULE += gnrc_mac
```

Comment:
Nope I meant something in the spirit of (please replace `x` and `y` with actual integers):

```Makefile
CFLAGS += -DGNRC_MAC_TX_QUEUE_SIZE=x -DGNRC_MAC_NEIGHBOR_COUNT=y # ...
```

Otherwise you just test the default configurations, though diverging configurations are far more interesting for testing